src/classes/Recipe.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable max-len */
+import Ingredient from './Ingredient';
+
+export default class Recipe {
+  constructor(recipeDetails) {
+    this.id = recipeDetails.id;
+    this.image = recipeDetails.image;
+    this.instructions = recipeDetails.instructions;
+    this.name = recipeDetails.name;
+    this.tags = recipeDetails.tags;
+    this.recipeIngredients = recipeDetails.ingredients;
+    this.processedIngredients = null;
+  }
+
+  processIngredients(dataBase) {
+    this.processedIngredients = this.recipeIngredients.map(ingredientDetails => {
```

Comment:
I wonder if reassigning recipeIngredients would be better here versus making another property? I could see how there could be confusion if we had two properties that are very similar things.

Reply:
I changed this class, we'll review it later